### PR TITLE
Makes sure that papers' SNPs are unique. Adds non-working rake task to make papers' SNPs unique.

### PR DESCRIPTION
**Pull request by philippbayer**
This should hopefully prevent #202 from happening again 

The Rake task doesn't work right now for some strange reason. It calls commit and everything but the paper remains "broken". I should drink more coffee.

(We can circumvent this issue by just storing all snps in a set instead of an array, but it's not really an array but a ActiveRecord::Associations::CollectionProxy)

### CODE REVIEW

**Comment by philippbayer** [comment]
Build fails because: Coverage has dropped by 4.40% since the last time 

I assume the 4% are added by the untested rake task? How do you even test rake tasks?

**Comment by tsujigiri** [comment]
I'm getting the feeling that the coverage varies between machines. On your machine, simplecov seems to have increased the coverage, which made it fail on TravisCI.

File: coverage/.last_run.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 51.41
+    "covered_percent": 55.81
```
Comment (tsujigiri) [inline coverage/.last_run.json]: Just revert that change for now and you should be good. Are you using the same Ruby version as TravisCI does?

**Comment by philippbayer** [comment]
Even with ```rvm use 2.2 ``` (after installing everything) I get the 55% coverage. Reverted it for now.

What I don't understand - the current .last_coverage file has 51%, my commit had 55% coverage. It complained that coverage dropped by 4%, but isn't it the other way around?

**Comment by tsujigiri** [comment]
If the file says 55% and the tests say 51%, the coverage dropped by 4%, no?

I just changed the Ruby version in the .travis.yml from "2.2" to "2.2.0" to see if TravisCI would use the latest patch-level instead of "2.2.0-p0", but it didn't. It still used the same patch level. The tests failed though because of a coverage drop. Nothing else changed. I would say, let's drop this for now. It's not really useful if it isn't deterministic, is it?

**Comment by philippbayer** [comment]
Yes, it does seem to be quite random!

**Comment by tsujigiri** [comment]
See #204

**Pull request merged**